src/scripts/ui/PlayerTradeComponent.tsx

```diff
@@ -34,6 +34,7 @@ import { TableView } from "./TableView";
 import { WarningComponent } from "./WarningComponent";
 
 const savedResourceFilters: Set<Resource> = new Set();
+const caravanTradesSortingState = { column: 0, asc: true };
```

Comment:
A minor thing, use `playerTradesSortingState` to be consistent?

Reply:
Done